templates/file.j2

```diff
@@ -46,12 +46,30 @@
   {%- for label in data.labels %}
   <a class="badge rounded-pill"
      href='https://github.com/leanprover-community/mathlib4/pulls?q=is%3Apr+is%3Aopen+label%3A"{{label.name}}"'
-     style="background-color:#{{label.color}};color:{{label.text_color}}">
+     style="background-color:#{{label.color}};color:{{label.color | text_color_of_color}}">
     {{label.name}}
   </a>
   {%- endfor %}
 </div>
 {% endif %}
+{% if data.status.mathlib4_sync_prs %}
+    <div class="alert alert-warning">
+        This file has open sync PRs:
+        <ul>
```

Comment:
I think this section would go well immediately under the "changes in mathlib4" heading. I plan to augment it with the commit information in a later date.